main.go

```diff
@@ -341,6 +350,18 @@ func (cmd *mainCmd) Run(args []string) error {
 	return multierr.Combine(errors...)
 }
 
+func checkGeneratedCode(f *ast.File) bool {
+	if ast.IsGenerated(f) {
+		return true
+	}
+	for _, comm := range f.Doc.List {
+		if strings.Contains(comm.Text, "@generated") {
+			return true
```

Comment:
perhaps output a verbose log here saying this file was skipped because it's generated so that it's easier to find out in the case of false positive cases that a file got skipped from patch.

Reply:
Good point! Added the log.